stable/nginx-ingress/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.34.3
+version: 1.34.4
```

As this is a new feature of the chart, let's bump the minor version.
```suggestion
version: 1.35.0
```

---

Thanks, I wasn't sure about that.